tests/collections/nlp/test_prompt_learning.py

```diff
@@ -71,8 +71,8 @@ def get_task_templates():
         "task_id_num": 0,
     }
     task_templates['task name B'] = {
-        "prompt_template": "<|VIRTUAL_PROMPT_0|>{question}<|VIRTUAL_PROMPT_1|>{answer}{extra}",
-        "prompt_template_fields": ['question', 'answer'],
+        "prompt_template": "<|VIRTUAL_PROMPT_0|> {question} <|VIRTUAL_PROMPT_1|> {answer}{extra}",
```

Comment:
why you add the space around the virtual tokens? If space is needed, virtual prompt should learn to be a space embedding?

Reply:
I was playing around with the unit tests to help diagnose a bug with the huggingface tokenizer. This change isn't needed, I just forgot to put it back. (Turns out there is a bug with the huggingface tokenizer, btw)